internal/namespaces/instance/v1/custom_runs.go

```diff
@@ -163,3 +170,48 @@ func customInstanceImageListRun(ctx context.Context, argsI interface{}) (i inter
 
 	return customImages, nil
 }
+
+// customInstanceServerTypeListRun transforms the server map into a list to display a
+// table of server types instead of a flat key/value list.
+func customInstanceServerTypeListRun(originalRun core.CommandRunner) core.CommandRunner {
+	return func(ctx context.Context, argsI interface{}) (interface{}, error) {
+
+		type customServerType struct {
+			Name            string        `json:"name"`
+			MonthlyPrice    *scw.Money    `json:"monthly_price"`
+			HourlyPrice     *scw.Money    `json:"hourly_price"`
+			LocalVolumeSize scw.Size      `json:"local_volume_size"`
+			CPU             uint32        `json:"cpu"`
+			GPU             *uint64       `json:"gpu"`
+			RAM             scw.Size      `json:"ram"`
+			Arch            instance.Arch `json:"arch"`
+		}
+
+		originalRes, err := originalRun(ctx, argsI)
```

Comment:
the originalRun is short: 7 lines, with error handling

Can't we just copy it here ? 
Passing `originalRun core.CommandRunner` adds complexity just to remove ~4 lines of code. 

https://github.com/scaleway/scaleway-cli/blob/e299f8177049f826f566bf78097092b2b43bcc6c/internal/namespaces/instance/v1/instance_cli.go#L1197

Reply:
Even if `originalRun` have is only 7 lines long it is 7 lines that we don't have to maintain.
The complexity of getting the original `Run` function (in `applyCustomRuns`) will be removed with the future `OverrideRun` method.